treeopt/treeOpt.py

```diff
@@ -257,7 +256,9 @@ def read_data(self, filename):
         :rtype: Numpy Array
         """
 
-        path = os.path.join(os.getcwd() + + os.sep + "treeOptData", filename + ".csv")
+        path = os.path.join(
+            os.getcwd() + +os.sep + "treeOptData", filename + ".csv"
```

Comment:
Not a big deal, but, you have a plus to much and `os.path.join` automatically adds the correct seperator if you do

```python
os.path.join(os.getcwd(), "teeOptData", filename + ".csv")
```